knife/lib/chef/chef_fs/knife.rb

```diff
@@ -131,6 +131,9 @@ def pattern_arg_from(arg)
           ui.error("Attempt to use relative path '#{arg}' when current directory is outside the repository path.")
           ui.error("Current working directory is '#{@chef_fs_config.cwd}'.")
           exit(1)
+        elsif !Dir.exist?(arg)
```

Comment:
does this break `knife download /pathname` when the path to download doesn't exist yet because knife hasn't created it?  (also this is probably used by more than just cookbook code)